langchain/src/chat_models/openai.ts

```diff
@@ -288,10 +288,10 @@ export class ChatOpenAI extends BaseChatModel implements OpenAIInput {
               if (part != null) {
                 innerCompletion += part.delta?.content ?? "";
                 role = part.delta?.role ?? role;
-                this.callbackManager.handleNewToken?.(
-                  part.delta?.content ?? "",
-                  this.verbose
-                );
+                // this.callbackManager.handleNewToken?.(
+                //   part.delta?.content ?? "",
+                //   this.verbose
+                // );
```

Comment:
@nfcampos -- there's some issues with using async callbacks for streaming i.e. async handleLLMNewToken in the LLM classes bc you're using createParser which is sync. What's the best workaround? I've commented out all the streaming callbacks and int tests that use streaming